locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/templates/customize/CustomizeTemplateOptions2.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.brooklyn.location.jclouds.templates.customize;
+
+// Has "2" in its name so JcloudsLocation can use it without having to use a fully qualified reference.
+public class CustomizeTemplateOptions2 {
```

Comment:
Unfortunate that `CustomizeTemplateOptions` is already taken. I don't like the "2" in the name, but not sure what to suggest that would be better (unless we can delete the existing interface `CustomizeTemplateOptions`).

I don't think the interface `JcloudsLocation.CustomizeTemplateOptions` will be mentioned in any persisted state, so deleting it is just about source-level compatibility with anything that uses it directly. I lean towards treating that as a part of our "internal" API, so deleting it. It should be easy to fix any downstream projects that then fail to compile.

We'd then use the nicer name of `CustomizeTemplateOptions` for this class.

@sjcorbett Thoughts?